docs/stylesheets/extra.css

```diff
@@ -103,3 +103,15 @@
   color: var(--md-code-fg-color);
   background-color: var(--md-code-bg-color);
 }
+
+.md-nav__title {
+  display: none;
+}
+
+.md-nav__link {
+  margin-top: 0.3em;
+}
+
+.md-nav--secondary .md-nav__link {
+  margin-top: 0.5em;
```

Comment:
Note this specific change was to prevent the right-side TOC from also being condensed.